tests/Shared/IdentityUtilities.cs

```diff
@@ -305,5 +305,41 @@ public static bool LifetimeValidatorThrows(DateTime? expires, DateTime? notBefor
         {
             throw new SecurityTokenInvalidLifetimeException("LifetimeValidatorThrows");
         }
+
+        public const string InvalidIssuer = null;
+
+        public static TokenValidationParameters InvalidIssuerAsymmetricTokenValidationParameters
+        {
+            get { return InvalidIssuerTokenValidationParameters(DefaultAsymmetricSigningToken); }
+        }
+
+        public static TokenValidationParameters InvalidIssuerTokenValidationParameters(SecurityToken securityToken)
+        {
+            return new TokenValidationParameters
+            {
+                AuthenticationType = DefaultAuthenticationType,
+                IssuerSigningToken = securityToken,
+                ValidAudience = DefaultAudience,
+                ValidIssuer = InvalidIssuer,
+                ValidateIssuer = false,
```

Comment:
You should set this to true.
This flag, bypasses validation of the Issuer.

You should have two tests one positive and one negative.
This will let you know that you have everything working right.